### PR TITLE
Espresso tests

### PR DESCRIPTION
**Pull request by mustafa01ali**

### CODE REVIEW

File: app/src/androidTest/java/com/mutualmobile/barricade/sample/MainActivityTest.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  @Test public void verifyEnablingBarricade() {
+    onView(withId(R.id.barricade_switch)).perform(click());
+    onView(withId(R.id.get_joke_button)).perform(click());
+    onView(withId(R.id.joke_text)).check(matches(withText(JOKE_FROM_BARRICADE)));
+  }
+
+  @Test public void verifyDisablingBarricade() {
+    onView(withId(R.id.get_joke_button)).perform(click());
+    onView(withId(R.id.joke_text)).check(matches(not(withText(JOKE_FROM_BARRICADE))));
```
Comment (tushar-nallan) [inline app/src/androidTest/java/com/mutualmobile/barricade/sample/MainActivityTest.java:51]: It is possible that we get the exact same joke from backend too. Only solution on the top of my head is to append a unique string like "- Barricade" to the joke returned by barricade. @mustafa01ali what do you think?

Comment (mustafa01ali) [reply app/src/androidTest/java/com/mutualmobile/barricade/sample/MainActivityTest.java:51]: Well, we can be more creative and come up with an original Chuck Norris joke.

**Review by tushar-nallan — APPROVED** [review]

File: app/src/androidTest/java/com/mutualmobile/barricade/sample/MainActivityTest.java (hunk 1 above)
Comment (tushar-nallan) [reply app/src/androidTest/java/com/mutualmobile/barricade/sample/MainActivityTest.java:51]: I literally spent 20 minutes for this and still got nothing. @suchoX Please help 😛

Comment (mustafa01ali) [reply app/src/androidTest/java/com/mutualmobile/barricade/sample/MainActivityTest.java:51]: lol, I added a new one

Comment (tushar-nallan) [reply app/src/androidTest/java/com/mutualmobile/barricade/sample/MainActivityTest.java:51]: haha nice 😄

**Pull request merged**